styleguide-app/Examples/Modal.elm

```diff
@@ -252,6 +331,7 @@ type Msg
     | SetShowContinue Bool
     | SetShowSecondary Bool
     | SetDismissOnEscAndOverlayClick Bool
+    | SetLongContent Bool
```

Comment:
Without a visible title the scroll hints conflict with the X:

![image](https://user-images.githubusercontent.com/348564/63257870-a0b83d80-c272-11e9-8f81-81d298b57fe1.png)

I'm not sure that should block this PR though.

Reply:
It can also overlap with the focus hint for the X when there is a title:

![image](https://user-images.githubusercontent.com/348564/63258040-11f7f080-c273-11e9-9e45-6d6ce3158aaf.png)

Again, I don't think this is a blocker.